tools/cli-bootstrap/cli.bootstrap.py

```diff
@@ -0,0 +1,562 @@
+#!/usr/bin/env python 
+
+import os
+import json
+import platform
+import argparse
+import sys
+import traceback
+
+# for readability
+from subprocess import call
+from subprocess import check_output
+from subprocess import check_call
+from subprocess import CalledProcessError
+
+from os import path
+from os import makedirs
+
+from string import find
+from urllib import urlretrieve
+
+
+# ROVER BASE #
+class RoverMods:
+    HEADER = '\033[95m'
```

Comment:
Does this stuff work in docker containers? I'm pretty wary of us putting any colors into stuff now, since our attempts elsewhere rarely work in docker, and often break basic text output and formatting.